README.md

```diff
@@ -215,6 +193,6 @@ module "landing_zone" {
 |------|-------------|
 | kms\_key\_arn | ARN of KMS key for SSM encryption |
 | kms\_key\_id | ID of KMS key for SSM encryption |
-| monitor\_iam\_access\_event\_bus\_arn | ARN of the Event Bus in the Audit account for IAM access monitoring notifications |
+| monitor\_iam\_access\_sns\_topic\_arn | ARN of the SNS Topic in the Audit account for IAM activity monitoring notifications |
```

Comment:
```suggestion
| monitor\_iam\_activity\_sns\_topic\_arn | ARN of the SNS Topic in the Audit account for IAM activity monitoring notifications |
```

Reply:
Also need to be changed in the code of course :-)

Reply:
I also like iam\_activity\_sns\_topic\_arn, but then I would also suggest to change monitor\_iam\_activity to iam\_activity

Reply:
Make sure this name matches the variable name here: https://github.com/schubergphilis/terraform-aws-mcaf-landing-zone/pull/86/files#diff-fb5f00e0f1a658d6cebc3873ccb73868880bd8575b1a8d9ea31f5371d23ce14bR90